docs/faq.md

```diff
@@ -44,3 +44,7 @@ $ kubectl  -n cattle-system exec $(kubectl --kubeconfig $KUBECONFIG -n cattle-sy
 New password for default administrator (user-xxxxx):
 <new_password>
 ```
+
+### I added an additional disk, but why can the disk not get detected?
```

Comment:
```suggestion
### I added an additional disk with partitions. Why is it not getting detected?
```

I'd suggest making the question specific (mention partitions) to match the answer.